### ISSUE TITLE
Upgrade bundled jQuery Mobile from 1.0a4.1

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#35]
```
Current release is 1.0b1


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 19 Jul 2011 at 6:48

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 19 Jul 2011 at 7:00
- Added labels: **Milestone-Release1.2.4**

**Comment by GoogleCodeExporter** [comment]
```
Before going to 1.0b1, need to make sure there is a solution for 
http://forum.jquery.com/topic/back-button-error
```

Original comment by `paul.beu...@gmail.com` on 19 Jul 2011 at 7:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2.4
```

Original comment by `paul.beu...@gmail.com` on 19 Aug 2011 at 4:30
- Changed state: **Fixed**
